### PR TITLE
scaffolder-backend-module-gitlab: add gitlab:issues:create custom action 

### PR DESCRIPTION
**Pull request by elaine-mattos**
## Hey, I just made a Pull Request!

This creates a new GitLab Scaffolder custom action that allows Gitlab users to create issues.  This is a quick summary of what was done:
- Added the action, its examples and tests;
- Added more helper functions and its tests;
- Added template example to the readme.md file;
- Used the @gitbeaker/rest library as the @gitbeaker/node has been deprecated;

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info]
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info]

### CODE REVIEW

File: plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabIssueAction.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        'Id of a discussion to resolve. Use in combination with "merge_request_to_resolve_discussions_of"',
+    })
+    .optional(),
+  epicId: z
+    .number({ description: 'Id of the linked Epic' })
+    .min(0, 'Valid values should be equal or greater than zero')
+    .optional(),
+  labels: z.string({ description: 'Labels to apply' }).optional(),
+  issueType: z
+    .string({
```
Comment (benjdlambert) [inline plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabIssueAction.ts:70]: I wonder if it's possible to use `nativeEnums` here instead?

https://github.com/colinhacks/zod#native-enums

**Review by benjdlambert — COMMENTED** [review]
This is looking pretty good at first glance, just a question about the enums and if we can do those a slightly different way so that they're in the jsonschema? :pray:

File: plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabIssueAction.ts (hunk 1 above)
Comment (elaine-mattos) [reply plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabIssueAction.ts]: Hi guys,

it's done. Please let me know what you think! :)

**Comment by elaine-mattos** [comment]
Hi @benjdlambert ,

I just received the notification of this PR being closed due to lack of activity. Is there anything here that still needs to be done on my part?
Please let me know! :)

**Comment by freben** [comment]
We've been away for vacations - reopening!

**Review by benjdlambert — APPROVED** [review]
This looks great! Thanks!

**Pull request merged**